src/main/java/Duke.java

```diff
@@ -1,24 +1,37 @@
+import java.io.*;
```

Comment:
Should you use a wildcard import (*) here? Would it be better if you listed out the needed packages only?